app/views/home/update.php

```diff
@@ -1,136 +1,191 @@
+                            <?php
               switch ($data['obat']['unit']) {
                 case "botol":
                   echo "
               <option value='botol' selected>botol</option>
               <option value='tablet'>tablet</option>
               <option value='kapsul'>kapsul</option>";
-                  break;
+                break;
                 case "tablet":
                   echo "
               <option value='botol'>botol</option>
               <option value='tablet' selected>tablet</option>
               <option value='kapsul'>kapsul</option>";
-                  break;
+                break;
                 case "kapsul":
                   echo "
               <option value='botol'>botol</option>
               <option value='tablet'>tablet</option>
               <option value='kapsul' selected>kapsul</option>";
-                  break;
+                break;
+                default:
+                echo "
+            <option value='botol'selected>botol</option>
+            <option value='tablet'>tablet</option>
+            <option value='kapsul'>kapsul</option>";
+                break;
               }
               ?>
-            </select>
-            <div class="invalid-feedback">
-              Wajib diisi
-            </div>
-          </div>
-        </div>
-        <div class="mt-2">
-          <label class="form-label" for="efek-samping">efek samping</label>
-          <textarea maxlength="65535" name="efek-samping" id="efek-samping" class="form-control" placeholder="efek samping" required><?= $data['obat']['efek_samping'] ?></textarea>
-          <div class="invalid-feedback">
-            Wajib diisi
-          </div>
-        </div>
-        <div class="mt-2">
-          <label class="form-label" for="indikasi">indikasi</label>
-          <textarea maxlength="65535" type="text" name="indikasi" id="indikasi" class="form-control" placeholder="indikasi" required><?= $data['obat']['indikasi'] ?></textarea>
-          <div class="invalid-feedback">
-            Wajib diisi
-          </div>
+                        </select>
+                        <?php if(!empty($data['errors']['stok'])) { ?>
+                        <div class="invalid-feedback ">
+                            <?php echo $data['errors']['stok']; ?>
+                        </div>
+                        <?php } ?>
+                        <?php if(!empty($data['errors']['unit'])) { ?>
+                        <div class="invalid-feedback ">
+                            <?php echo $data['errors']['unit']; ?>
+                        </div>
+                        <?php } ?>
+                    </div>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="efek-samping">efek samping</label>
+                    <textarea maxlength="65535" name="efek_samping" id="efek_samping"
```

Comment:
`for` di label tidak sesuai dengan `id` di input.
input tidak bisa fokus ketika klik label